Dockerfile

```diff
@@ -1,21 +1,21 @@
 FROM amazonlinux:2.0.20200722.0
 
 ARG STACK_VERSION=2.5.1
+ARG USER=haskell
 
-ENV LANG=C.UTF-8 \
-  LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+ENV PATH="/home/$USER/.local/bin:$PATH"
 
 RUN yum update -y \
+  && amazon-linux-extras install postgresql11 \
```

Comment:
This was a command rather than a list of packages to install, so I moved it to actually be run as a command.